src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+type Props = {
+  name: string | undefined,
+  email: string | undefined,
+};
```

Comment:
don't repeat yourself
```suggestion
type Props = {
  user: User;
};
```